### PR TITLE
feat: add google analytics

### LINKED ISSUE #68: Integrate Google Analytics
to see incoming traffic and unique view count

### PR DESCRIPTION
**Pull request by bombnp**
Fixes #68   
  
now we can track how many people come in here

### CODE REVIEW

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+            dangerouslySetInnerHTML={{
+              __html: `
+              window.dataLayer = window.dataLayer || [];
+              function gtag(){dataLayer.push(arguments);}
+              gtag('js', new Date());
+              gtag('config', '${GA_MEASUREMENT_ID}');
+              `,
+            }}
+          />
+          <script src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script>
```
Comment (new5558) [inline src/pages/_document.tsx]: Why do we need this line?

Comment (bombnp) [reply src/pages/_document.tsx]: wait i overcopied lol

Comment (bombnp) [reply src/pages/_document.tsx]: removed

**Review by new5558 — APPROVED** [review]
LGTM

**Pull request merged**